source/guides/clear/mixer.rst

```diff
@@ -284,79 +312,107 @@ set to get a smaller kernel image, which will also be faster to load.
+bundle from Example 2, using `ister`_.
 
-   Repeat steps 1-3 to add the upstream :command:`curl` bundle to the mix:
+#. Change directory into your mix.
+
+#. Configure image. Edit the ister configuration file for your image to
+   include all of the bundles you want pre-installed in the image.
+   If this is the first time creating an image, first get a copy of the
+   :file:`release-image-config.json` template file:
 
    .. code-block:: bash
 
-      mixer bundle add curl
-      mixer build bundles
-      mixer build update
+      curl -O https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json
+
+   For this example, make the following revisions
+   :file:`release-image-config.json`:
 
-   Build optional delta-packs, which helps reduce client update time:
+   * Set root partition size to "5G"
+   * Replace the "kernel-native" bundle with "kernel-kvm".
+   * Set the version to 10 (as an integer).
+
+   .. note::
+      When creating an image, select a subset of the bundles that are part of your mix. All the bundles that are *not* part of this subset are available for consumers of that image to install afterwards via swupd.
 
    .. code-block:: bash
+      :linenos:
+      :emphasize-lines: 5,11-12
 
-      mixer build delta-packs --from 10 --to 20
+      {
+       "DestinationType" : "virtual",
+        "PartitionLayout" : [ { "disk" : "release.img", "partition" : 1, "size" : "32M", "type" : "EFI" },
+                                  { "disk" : "release.img", "partition" : 2, "size" : "16M", "type" : "swap" },
+                                  { "disk" : "release.img", "partition" : 3, "size" : "5G", "type" : "linux" } ],
+        "FilesystemTypes" : [   { "disk" : "release.img", "partition" : 1, "type" : "vfat" },
+                                  { "disk" : "release.img", "partition" : 2, "type" : "swap" },
+                                  { "disk" : "release.img", "partition" : 3, "type" : "ext4" } ],
+        "PartitionMountPoints" : [ { "disk" : "release.img", "partition" : 1, "mount" : "/boot" },
+                                       { "disk" : "release.img", "partition" : 3, "mount" : "/" } ],
+        "Version": 10,
+        "Bundles": ["kernel-kvm", "os-core", "os-core-update"]
+       }
 
-   Refresh your \http://localhost site to see the update content for
-   mix version 20.
+#. Build the image.
 
-   Look in ~/mixer/update/www/<mix version> to see the update content in your
-   workspace.
+   .. code-block:: bash
+
+      sudo mixer build image
 
-Example 3: Deploy updates to target
+   The output from this step will be :file:`release.img`, which is a live
+   image.
+
+Example 5: Deploy updates to target
 ===================================
 
-The image created in Example 2 is directly bootable in QEMU. In this example,
+The image created in Example 4 is directly bootable in QEMU. In this example,
 we'll boot the image from Example 2 to verify it, and update the image from
```

Comment:
Line 414: 

"Example 2" should be "Example 4"

Reply:
Hmmm. My last commit shows Example 4.